vulture/core.py

```diff
@@ -90,12 +89,16 @@ def read_file(filename):
 
 
 def _get_suitable_whitelists(defined_imports):
-    all_whitelists = set(pkg_resources.resource_listdir('vulture',
-                                                        'whitelists'))
-    defined_imports = set([imp + '.py' for imp in defined_imports])
-    common = sorted(all_whitelists.intersection(defined_imports))
-    whitelist_paths = [os.path.join('whitelists', name) for name in common]
-    return whitelist_paths
+    wh_names = []
+    defined_imports = ['wh_' + imp for imp in defined_imports]
+    wh_mods = __import__('whitelists', globals(), locals(), defined_imports, 1)
```

Comment:
Thinking about this some more it seems a little strange to import a module only to convert it back to text. Maybe it's better to just use pkgutil.get_data(). If a defined import can't be loaded by pkgutil.get_data() the function returns None, and we try the next defined import.